src/OpenTelemetry.Instrumentation.Process/ProcessMetrics.cs

```diff
@@ -57,5 +58,74 @@ static ProcessMetrics()
     /// <param name="options">The options to define the metrics.</param>
     public ProcessMetrics(ProcessInstrumentationOptions options)
     {
+        // TODO: change to ObservableUpDownCounter
+        // MeterInstance.CreateObservableGauge(
+        //    $"process.cpu.utilization",
+        //    () => CurrentProcess.TotalProcessorTime.TotalMilliseconds / (Environment.ProcessorCount * (DateTime.Now - CurrentProcess.StartTime).Milliseconds),
+        //    unit: "1",
+        //    description: "Difference in process.cpu.time since the last measurement, divided by the elapsed time and number of CPUs available to the process.");
+
+        // What type should be used for the label?
+        // labels
+        // state, if specified, SHOULD be one of: system, user, wait. A process SHOULD be characterized either by data points with no state labels, or only data points with state labels.
+        // IEnumberable,
+        // List<Measurement<long>>
+        // wait time = totalprocessortime - Process.PrivilegedProcessorTime - Process.userProcessorTime
+
+        if (options.ExpandOnCpuStates == true)
+        {
+           MeterInstance.CreateObservableCounter(
+           $"process.cpu.time",
```

Comment:
consider adding link to the relevant OTel semantic convention doc.

Reply:
added link to the spec